localstack/services/es/es_api.py

```diff
@@ -23,6 +23,144 @@ def error_response(error_type, code=400, message='Unknown error.'):
     return response, code
 
 
+def get_domain_config(domain_name):
+    return {
+        'DomainConfig': {
```

Comment:
Wondering if this is the most "minimal" hardcoded domain config that we can return. Some of the configurations appear to be fairly specific to your scenario (e.g., `AdvancedOptions`), so it is unclear how well they generalize to arbitrary use cases.

Also, there are 9 `'Status'` dicts that are very similar in this configuration - would it make sense to have a small parameterizable utility function that returns a corresponding `Status` dict?

Reply:
I copied the JSON from what's documented here:
https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-configuration-api.html#es-configuration-api-actions-describeelasticsearchdomainconfig

I can get rid of some of the more specific options if it helps.

I'll get rid of the duplicated `'Status'` dicts with a small function as well.